docs-2.0/3.ngql-guide/10.tag-statements/2.drop-tag.md

```diff
@@ -0,0 +1,29 @@
+# DROP TAG
+
+```ngql
+DROP TAG [IF EXISTS] <tag_name>
+```
+
+`DROP TAG` drops a tag with the given name in a graph space. You must have the `DROP` privilege for the graph space. (TODO: privilege doc) To drop a tag in a specific graph space, you must use the graph space first.
+
+Before you drop a tag, make sure that the tag does not contain any indexes or TTLs. If a tag contains any indexes or TTLs, a conflict error occurs when you drop it.
```

Comment:
> If a tag contains any indexes or TTLs, a conflict error occurs when you drop it.

Use "Otherwise" instead of "If a tag contains any indexes or TTLs"?

Repeating what we just wrote can be inefficient and boring.

Reply:
Otherwise is not an allowed word in standard technical English. I want to make the doc clear.